package.json

```diff
@@ -31,14 +31,14 @@
     "posttest": "npm run lint"
   },
   "devDependencies": {
-    "async-iterators": "^0.2.2",
     "eslint": "^3.11.1",
     "eslint-config-loopback": "^6.0.0",
     "mocha": "^2.1.0",
     "should": "^8.0.2"
   },
   "dependencies": {
     "async": "~1.0.0",
+    "async-iterators": "^0.2.2",
```

Comment:
I simply moved this from a devDep to a dep. It's actually not needed as a full on dep anymore with your suggested changes for deleteAll, so removing this as it's unused now.